next/components/contact-form.tsx

```diff
@@ -51,9 +53,10 @@ export function ContactForm() {
           </button>
         </div>
       )}
-      {!isSubmitted && (
+      {!isSubmitSuccessful && (
         <form
-          onSubmit={void handleSubmit(onSubmit)}
```

Comment:
This fix made eslint happy, but also broke the form :)

So I added the ignore line instead.